src/overlay_01_02204ED8.c

```diff
@@ -0,0 +1,215 @@
+#include "assert.h"
+#include "encounter.h"
+#include "field_follow_poke.h"
+#include "field_map_object.h"
+#include "fieldmap.h"
+#include "filesystem.h"
+#include "heap.h"
+#include "overlay_01.h"
+#include "overlay_01_021EDAFC.h"
+#include "overlay_02.h"
+#include "player_data.h"
+#include "scrcmd.h"
+#include "script.h"
+#include "task.h"
+#include "unk_0205CB48.h"
+#include "unk_0205FD20.h"
+
+typedef struct UnkStruct_02204EFC {
```

Comment:
Define the struct in overlay_01_02204ED8.h

Reply:
actually, where is that file?

Reply:
> Define the struct in overlay_01_02204ED8.h

Is that necessary if it's only used in this file?

Reply:
I suppose not, but where is overlay_01_02204ED8.h?

Reply:
Not needed then since the script command functions have their own header file and everything else is static.

Reply:
what about `ov01_02204ED8`?

Reply:
Oops okay this isn't static but it's already in `overlay_01.h`.

Reply:
`overlay_01.h` is a relic left from before `overlay_01.s` was split up, we're trying to phase it out so it would be very much appreciated if you can make a new header file even if it's just for this one function